frontend/interfaces/software.ts

```diff
@@ -276,6 +277,7 @@ export interface ISoftwareInstallResult {
   post_install_script_output: string;
   created_at: string;
   updated_at: string | null;
+  automatic_install: boolean;
```

Comment:
@lucasmrod Will this return? or is it just for post

Reply:
Just for the `POST`. Because what makes the installer automated is the fact that it has a policy associated to it. The UI already renders the "Automatic install" using the `automatic_install_policies` field.